src/signals/incident/components/IncidentForm/index.js

```diff
@@ -33,7 +33,9 @@ class IncidentForm extends Component {
       return null
     }
 
-    const loading = get(props, props.postponeSubmitWhenLoading)
+    const loading = Array.isArray(props.postponeSubmitWhenLoading)
+      ? props.postponeSubmitWhenLoading.some((prop) => get(props, prop))
+      : get(props, props.postponeSubmitWhenLoading)
     if (loading !== prevState.loading) {
```

Comment:
The code becomes overly complex this way. I would prefer to always use an array.